src/Simplify.elm

```diff
@@ -10425,6 +10449,43 @@ operationDoesNotChangeResultOfOperationCheck =
     }
 
 
+{-| When a function is given two equal arguments it will return the last argument.
+
+    f a a
+    --> a
+
+Examples:
+
+    Dict.union dict dict
+    --> dict
+
+    Set.union set set
+    --> set
+
+-}
+whenArgumentsAreEqualReturnLastCheck : IntoFnCheck
```

Comment:
Maybe make clear in the name that this only applied two the first 2 arguments. Something like `onTwoEqualArgumentsReturnsLast`

Reply:
Any reason why you chose to name it `onEqualFirstArgumentsReturnLastCheck` (not "two" and not "with")?

Reply:
`on` because that's what you suggested at first before editing (and I thought that `on` was kind of the new naming convention, not sure what's best :shrug: ). And I wanted to name it `onTwoFirst...` or `onFirstTwo...` but that sounded more confusing than helpful.

I'll name it `withTwoEqualArgumentsReturnsLast`. Should I suffix it with "check"?

Reply:
Yeah, with -check